### ISSUE TITLE
How to interpret reconstruction loss of VAE

### ISSUE THREAD

**Issue by imtiendat0311** [welch-lab/MichiGAN#2]
Hi i'm Dat Nguyen CS student who currently research on VAE and detect underlying pattern of single cell data. I did review your code and notice you have similar way of calculate loss of VAE.

Pytorch code

```pyton
reproduction_loss = nn.functional.mse_loss(x_hat, x, reduction='none').sum(dim=1).mean()
KLD = torch.mean(-0.5 * torch.sum(1 + log_var - mean **
                     2 - log_var.exp(), dim=1), dim=0)

```
But the problem for this is reproduction_loss is too big since we only reduce it to mean ( batch size ) 

So my question is how could you interpret those loss and make sure that the model going to correct direction.
